### DISCUSSION TITLE
Why am I seeing this dark spot on my image?

### DISCUSSION THREAD

**Discussion by btwachter** [IBEXImagingCommunity/ibex_imaging_knowledge_base#41] [Q&A] [answered]
There is a dark spot in my image not on the sample... Nuclear staining on a confocal acquired thymus sample. Thank you!

![DarkSpot_resize](https://user-images.githubusercontent.com/127550776/224860311-4a4a52c5-26d3-4a1b-a7f7-b652b355d785.jpg)

**Comment by radtkea** [comment, marked as answer]
Hi Brianna,

Sorry you are encountering this problem. I suspect you are using an oil objective and there is a bubble in your immersion oil. The bubble creates the shadow/dark circle in your image.

Look closely at the oil interface between your specimen and the objective. I suspect you will be able to see a bubble with your naked eye. If so, then remove the oil and apply again, taking care to not add bubbles. I frequently remove/nudge bubbles using a 10 ul pipet tip after applying immersion oil and before imaging.

Best of luck! Thanks for sharing your image!
Andrea
